tests/workflows/snvphylnfc.nf.test

```diff
@@ -145,4 +145,28 @@ nextflow_workflow {
 
     }
 
+    test("disabling snv density filtering") {
+
+        when {
+            params {
+                input = "https://raw.githubusercontent.com/phac-nml/snvphylnfc/dev/assets/samplesheet.csv"
+                refgenome = "https://raw.githubusercontent.com/phac-nml/snvphylnfc/dev/assets/reference.fasta"
+                outdir = "results"
+                skip_density_filter = true
+            }
+            workflow {}
+        }
+
+        then {
+            assert workflow.success
+            assert path("$launchDir/results").exists()
+
+            // check that no "_filtered_density.txt" files exist:
+            assert path("$launchDir/results/consolidate/SAMPLE1_filtered_density.txt").exists() == false
+            assert path("$launchDir/results/consolidate/SAMPLE2_filtered_density.txt").exists() == false
+            assert path("$launchDir/results/consolidate/SAMPLE3_filtered_density.txt").exists() == false
```

Comment:
Something else you could also add, if density filtering is skipped, then the concatenated invalid positions file should be missing strings like `#Calculation and writing of high density regions has completed`. If it is not skipped, it should contain those strings.

Could you add asserts in the test both with and without density filtering to check for this?

Reply:
Added in 3891cce31a1881bf07c4f52b68cd35eb9b96489b